pkg/kubelet/qos/types.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package qos
+
+// Tier defines the supported qos classes of Pods/Containers.
+type Tier string
```

Comment:
I hate stuttering, so i dont want a qos.QosClass object... so qos.Class or
qos.Tier

On Fri, Jun 24, 2016 at 5:07 PM, Buddha Prakash <notifications@github.com>
wrote:

> In pkg/kubelet/qos/types.go
> <https://github.com/kubernetes/kubernetes/pull/27749#discussion_r68462939>
> :
>
> > +you may not use this file except in compliance with the License.
> > +You may obtain a copy of the License at
> > +
> > +    http://www.apache.org/licenses/LICENSE-2.0
> > +
> > +Unless required by applicable law or agreed to in writing, software
> > +distributed under the License is distributed on an "AS IS" BASIS,
> > +WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +See the License for the specific language governing permissions and
> > +limitations under the License.
> > +*/
> > +
> > +package qos
> > +
> > +// Tier defines the supported qos classes of Pods/Containers.
> > +type Tier string
>
> Haha.
>
> But lets use Class consistently in the code base. I will make changes to
> describe.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27749/files/3aa41d505486e9015f7f63ab2db3b2e0f2a4e51e#r68462939>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF8dbIf0V4YOKtHb84VGelE5GH_7KY5Gks5qPEcQgaJpZM4I6SZg>
> .
>